paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1499,6 +1499,10 @@ def check_if_instance_is_done(
                     instance, service, cluster
                 )
             )
+        elif e.body and "Temporary" in e.body:
```

Comment:
thought on subclassing ApiFailure so that we don't need to do a potentially brittle check for what's in the exception message and can instead check what the exception actually was?

Reply:
I couldn't quickly untangle the autogenerated oapi client-side handling of exceptions here, as they all come from a single auto-generated [ApiException](https://github.com/Yelp/paasta/blob/master/paasta_tools/paastaapi/exceptions.py#L101-L126) class.

I've gone with a slightly-less-brittle option of checking a specific non-standard error status code, rather than trying to check text.  Thoughts?

Reply:
rip

but i do prefer this over coupling the exception handling with the exception message